tensorflow_datasets/core/image_visualizer.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+"""Visualizer for image datasets.
```

Comment:
Rather than a module, it should be a class and have `SupervisedVisualizer` and `ObjectVisualizer` depend on it:

```
class Visualizer(object):

  def match(self,...)

   ...
```